scripts/add-cluster.sh

```diff
@@ -363,6 +356,12 @@ if [[ -n `oc get secret -n ${CLUSTER_JOIN_TO_OPERATOR_NS} ${OC_ADDITIONAL_PARAMS
 fi
 oc create secret generic ${SECRET_NAME} --from-literal=token="${SA_TOKEN}" --from-literal=ca.crt="${SA_CA_CRT}" -n ${CLUSTER_JOIN_TO_OPERATOR_NS} ${OC_ADDITIONAL_PARAMS}
 
+# Since MULTI_MEMBER variable is appended at the end of kubernetes object name for toolchaincluster resource,
+# let's set a "member id" if not provided and for names which il be truncated, so that we are sure that those object names will end with an alphanumerical char.
+if [ -z "$MULTI_MEMBER" ] && [ ${#JOINING_CLUSTER_NAME} -ge 63 ];
+then
+      MULTI_MEMBER=1
+fi
```

Comment:
this should be rather moved down and change to something like:
```
CLUSTERNAME_LENGTH_TO_KEEP=$((62-CLUSTERNAME_LENGTH_TO_REMOVE))

if [ -z "$MULTI_MEMBER" ] && [ ${#JOINING_CLUSTER_NAME} -ge ${CLUSTERNAME_LENGTH_TO_KEEP} ];
```
otherwise, it won't handle the cases when the cluster name has for example 60 chars.

Reply:
hmmm you're right ! I have to compare with `CLUSTERNAME_LENGTH_TO_KEEP`.

but why `62` in `62-CLUSTERNAME_LENGTH_TO_REMOVE` ? If I'm not mistaken `63` should work fine!

Reply:
because you don't know if you will add the `1` suffix or not :wink:

Reply:
Ok, because moving it down the part: `CLUSTERNAME_FIXED_PART="${JOINING_CLUSTER_TYPE_NAME}-${MULTI_MEMBER}"` may contain an empty `MULTI_MEMBER` so we need to keep in account of that. But when it's not empty we are "loosing" one char i think.

So probably we need to keep `63` and add a fixed id when we count the fixed part:
```
CLUSTERNAME_FIXED_PART="${JOINING_CLUSTER_TYPE_NAME}-1"
```

Reply:
makes sense, it's not super important to keep that, but just to be sure.

Thanks for pointing out this issue, please check [bba01ad](https://github.com/codeready-toolchain/toolchain-cicd/pull/83/commits/bba01ad78bcc530c88cfe49c04005a578158bd7c)

I'll trigger another run of e2e test job in `toolchain-e2e` and `openshift/release` , just to be sure both OCP and hypershift pipelines are still working fine.